specification/attestation/data-plane/Microsoft.Attestation/preview/2022-09-01-preview/attestation.json

```diff
@@ -0,0 +1,1206 @@
+      "properties": {
+        "x-ms-policy-certificates": {
+          "description": "SHA256 Hash of the binary representation certificate which was added or removed",
+          "$ref": "../../../../../common-types/rfcs/rfc7517.json#/definitions/JSONWebKeySet",
+          "x-ms-client-name": "PolicyCertificates"
+        }
+      }
+    },
+    "PolicyCertificatesModifyResponse": {
+      "type": "object",
+      "description": "The response to an attestation policy management API",
+      "properties": {
+        "token": {
+          "description": "An RFC7519 JSON Web Token structure whose body is a PolicyCertificatesModificationResult object.",
+          "$ref": "#/definitions/JsonWebToken"
+        }
+      }
+    },
+    "PolicyCertificatesModificationResult": {
+      "type": "object",
+      "description": "The result of a policy certificate modification",
+      "properties": {
+        "x-ms-certificate-thumbprint": {
+          "description": "Hex encoded SHA1 Hash of the binary representation certificate which was added or removed",
+          "type": "string",
+          "x-ms-client-name": "CertificateThumbprint"
+        },
+        "x-ms-policycertificates-result": {
+          "description": "The result of the operation",
+          "type": "string",
+          "x-ms-client-name": "CertificateResolution",
+          "enum": [
+            "IsPresent",
+            "IsAbsent"
+          ],
+          "x-ms-enum": {
+            "name": "CertificateModification",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "IsPresent",
+                "description": "After the operation was performed, the certificate is in the set of certificates."
+              },
+              {
+                "value": "IsAbsent",
+                "description": "After the operation was performed, the certificate is no longer present in the set of certificates."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "StoredAttestationPolicy": {
+      "type": "object",
+      "properties": {
+        "AttestationPolicy": {
+          "description": "Policy text to set as a sequence of UTF-8 encoded octets.",
+          "type": "string",
+          "format": "base64url"
+        }
+      }
+    },
+    "TcbBaseline": {
+      "type": "object",
+      "description": "TEE specific Tcb baseline used in attestation, and baseline details including baseline identifier, release date and minimum required software version",
+      "properties": {
+        "tcbEvaluationDataNumber": {
+          "description": "A monotonically increasing sequence number changed when Intel updates the content of the TCB evaluation data set. SGX TEE specific property",
+          "type": "string"
+        },
+        "tcbReleaseDate": {
+          "description": "Date and time when the Tcb is released. SGX TEE specific property",
```

Comment:
If this is a string, how do you reflect the customers time format preferences? Would it make sense to make this a [`"format":"dateTime"`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#data-types) field?

Reply:
Clarification: `"format": "date-time"`.

Reply:
Thanks! Updated.